client/src/components/publications/form/ImportSuccessPage.js

```diff
@@ -80,7 +80,7 @@ const ImportSuccessPage = ({ closeModal }) => {
     let checkedPublications = publications.filter(
       (pub, idx) => publicationsToImport[idx],
     );
-    for (let i = 0; i < checkedPublications.length; i++) {
+    for (let i = 0; i < checkedPublications.length; i += 1) {
```

Comment:
Can we hold onto this rule? https://github.com/Researchify/Researchify/blob/main/api/.eslintrc.json#L16 we're doing the same for the API. For consistency and also I kinda prefer `++` rather than `+=`.

Reply:
Sure I'll turn this rule off